src/overlays/actors/ovl_En_Ot/z_en_ot.c

```diff
@@ -41,128 +81,1099 @@ const ActorInit En_Ot_InitVars = {
+                        break;
+                }
+                break;
+            }
+            break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5CBEC.s")
+        case 1:
+        case 2:
+        case 3:
+            break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5CC88.s")
+        case 4:
+        case 5:
+        case 6:
+            if (func_80147624(globalCtx)) {
+                switch (globalCtx->msgCtx.unk11F04) {
+                    default:
+                    case 0x1068:
+                    case 0x106B:
+                    case 0x106C:
+                        func_80B5D648(this, globalCtx);
+                        break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5CCA0.s")
+                    case 0x1069:
+                        this->unk_32C |= 4;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5CCF4.s")
+                    case 0x106A:
+                        func_80B5CE6C(this, globalCtx);
+                        break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5CD40.s")
+                    case 0x10A5:
+                        func_801477B4(globalCtx);
+                        func_80B5CE6C(this, globalCtx);
+                        break;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5CE6C.s")
+                    case 0x106D:
+                        func_801477B4(globalCtx);
+                        func_80B5C154(this, globalCtx);
+                        break;
+                }
+            }
+            break;
+    }
+}
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5CEC8.s")
+s32 func_80B5D37C(GlobalContext* globalCtx, EnDno_ActorUnkStruct* arg1) {
+    s32 pad;
+    EnOt* temp_s0 = (EnOt*)arg1->unk_48;
+    f32 sp24;
+    f32 sp20;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5D114.s")
+    sp24 = Math_CosS(temp_s0->actor.world.rot.x * -1) * temp_s0->actor.speedXZ;
+    sp20 = gFramerateDivisorHalf;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5D160.s")
+    temp_s0->actor.velocity.x = Math_SinS(temp_s0->actor.world.rot.y) * sp24;
+    temp_s0->actor.velocity.y = Math_SinS(temp_s0->actor.world.rot.x * -1) * temp_s0->actor.speedXZ;
+    do {
+        temp_s0->actor.velocity.z = Math_CosS(temp_s0->actor.world.rot.y) * sp24;
+        temp_s0->unk_330.x += (temp_s0->actor.velocity.x * sp20) + temp_s0->actor.colChkInfo.displacement.x;
+        temp_s0->unk_330.y += (temp_s0->actor.velocity.y * sp20) + temp_s0->actor.colChkInfo.displacement.y;
+    } while (0);
+    temp_s0->unk_330.z += (temp_s0->actor.velocity.z * sp20) + temp_s0->actor.colChkInfo.displacement.z;
 
-#pragma GLOBAL_ASM("asm/non_matchings/overlays/ovl_En_Ot/func_80B5D37C.s")
+    return false;
+}
 
+#ifdef NON_MATCHING
+// Math_SmoothStepToS calls. I fixed this same pattern in
+// another file but I can't find it.
```

Comment:
Could you make this comment more helpful to the next person to look at this function?

Reply:
I'm not really sure what else I would write? How could I make it more helpful?

Reply:
Math_SmoothStepToS calls what? That part isn't clear to me

Reply:
There's only 3 calls in the function and it's a small function...

Reply:
Yeah I see where the 3 `Math_SmoothStepToS` is used. I still don't quite understand what this comment is saying. Are you calling the function? Are you calling the arguments? Are you calling a return? What pattern are you referring to? The fact that there are 3 `Math_SmoothStepToS`? Or does it have something to do with how the arguments are used? Or is it related to the temp and the way it's casted? 

There's a lot of ways to interpret this sentence. `Math_SmoothStepToS calls` feels like an incomplete sentence. I hope that's able to clear up my comment!